docker/README.md

```diff
@@ -0,0 +1,9 @@
+
+Rosetta/PyRosetta reference Docker recipes
+==========================================
+
+Collection of various recipes to serve as example for setting up Rosetta/PyRosetta build environment
```

Comment:
I think this should make clear that these recipes set up the environment to build (Py)Rosetta, but don't actually install/build (Py)Rosetta within it. (I was expecting them to include building Rosetta, given that the official images include Rosetta pre-installed.)

I'd probably also recommend adding a pointer to info about how you'd install (Py)Rosetta in the container. (Which may be as simple as "clone the repo & run scons", but I'd recommend being explicit.

Reply:
@roccomoretti Thank you for you thoughts! I will update docs a bit to pointers to initiate builds. Reason why this is not in the container (why this is a "reference" container not a recipe for our official images) is because in general cloning Rosetta repo and building _inside_ container is not good approach due it size. Instead it best to mount already cloned repo from host and build. (that how out official images is produced: it use multi stage process where binaries build by one container and then used in the final one to produce final image)